### PR TITLE
[CI:DOCS] Workaround quay.io image build failure

### PR DESCRIPTION
**Pull request by cevich**
Ref: [quay.io build failure](https://quay.io/repository/skopeo/ci/build/3992488d-ab5b-430a-856e-10a382c4b7cf)

Signed-off-by: Chris Evich <cevich@redhat.com>

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,4 +1,8 @@
-FROM fedora
+# Normally it's illadvised to use a FQDN, but this image is auto-built
```
Comment (mtrmac) [inline Dockerfile:1]: Personally curious: Is it? We typically warn against short names (acknowledging that they are convenient.)

File: Dockerfile [hunk 2]
```diff
@@ -1,4 +1,8 @@
...
+# on quay.io which has no setting to specify a non-docker default registry
+# or build-args.
+ARG REGISTRY=registry.fedoraproject.org/
+FROM ${REGISTRY}fedora:latest
```
Comment (mtrmac) [inline Dockerfile:5]: Absolutely non-blocking: Do we expect anyone to override the variable? Just hard-coding `registry.fedoraproject.org/fedora:latest` might be good enough.

**Comment by rhatdan** [comment]
LGTM

**Comment by rhatdan** [comment]
@mtrmac @vrothberg PTAL

**Review by mtrmac — COMMENTED** [review]
I can’t see the failure log, but this looks clearly fine.

Please merge as necessary.

File: Dockerfile (hunk 1 above)
Comment (cevich) [reply Dockerfile:1]: It's what I've been told/read as a best-practice.  It's more portable to use the short-name in the `Dockerfile` and rely on the build-host to configure the appropriate registry/registries within it's context.  However in practice, esp. with these CI-related Dockerfiles and repository auto-builders (like quay and docker hub), it's often easier to just jam the FQIN in and forget about it.

File: Dockerfile (hunk 2 above)
Comment (cevich) [reply Dockerfile:5]: TBH I didn't/don't know who/what uses this Dockerfile, so I made the more conservative choice.  I have no problem with "good enough" if that's what it is :grin:

**Comment by rhatdan** [comment]
I believe make uses this Dockerfile to build an image to run tests with.

**Comment by cevich** [comment]
> I believe make uses this Dockerfile to build an image to run tests with.

Correct.  Though rather than complicate things further, I think we can get away with hard-coding a non-docker-hub FQIN for now.  If an issue is raised, I'm happy to go the build-arg route.  Otherwise, I think @mtrmac is right, the vast majority of users and robots will never care.

**Pull request merged**